<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customrecursiontrip</class>
 <widget class="QMainWindow" name="customrecursiontrip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>300</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Final receipt:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recursive_pushButton_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>550</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Finish</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>90</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="recursive_comboBox_custom"/>
     </item>
     <item>
      <widget class="QListWidget" name="recursive_listWidget_custom"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Stadium: </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="recursive_listWidget_custom_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>651</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="recursive_pushButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>115</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start / Next city</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Souvenirs:                         </string>
    </property>
   </widget>
   <widget class="QListWidget" name="recursive_listWidget_custom_6">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>320</y>
      <width>371</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose teams to travel to:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>530</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total miles:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>521</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="recursive_listWidget_custom_3"/>
     </item>
     <item>
      <widget class="QListWidget" name="recursive_listWidget_custom_5"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>290</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected items:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>550</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="recursive_listWidget_custom_4"/>
     </item>
     <item>
      <widget class="QListWidget" name="recursive_listWidget_custom_7"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="recursive_pushButton_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>115</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Trip</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>949</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
